docs/integrations/nuxt.md

```diff
@@ -82,12 +82,12 @@ export default config
 or modify/extend it:
 
 ```ts
-import { mergeConfigs } from '@unocss/core'
+import { mergeConfigs } from 'unocss'
```

Comment:
UnoCSS pkg exports a lot of information. I think we don't need to change the source to know where the configuration is imported from.
```suggestion
import { mergeConfigs } from '@unocss/core'
```